myTrip/photo/models.py

```diff
@@ -42,27 +50,57 @@ def get_by_trip_id(trip_id):
         Args:
             trip_id (int): trip id.
         Returns:
-            QuerySet<Photo>: QuerySet of Photo.
+            QuerySet<Photos>: QuerySet of Photos.
         """
         photos = Photo.objects.filter(trip_id=trip_id)
         return photos
 
     @staticmethod
+    def get_by_photo_and_user_id(photo_id, user_id):
```

Comment:
get_by_photo_and_user_id and get_by_trip_id_and_checkpoint_id merge to one method